RecoHGCal/TICL/python/iterativeTICL_cff.py

```diff
@@ -22,14 +22,20 @@
 pfTICL = _pfTICLProducer.clone()
 ticlPFTask = cms.Task(pfTICL)
 
-iterTICLTask = cms.Task(ticlLayerTileTask
-    ,ticlTrkEMStepTask
+ticlIterations = cms.Task(
```

Comment:
For symmetry reasons, I would keep the `Task` at the end of the python label.

Reply:
OK, done.